### PR TITLE
Updates selected rows immediately on init

### PR DESCRIPTION
**Pull request by samschooler**
The table of selected elements currently only updates after a row is selected. This fixes this.

### CODE REVIEW

File: MultiSelect.js [hunk 1]
```diff
@@ -3,15 +3,16 @@ import { ListView, FlatList, SectionList } from 'react-native';
...
 
     this.renderRow = this.renderRow.bind(this);
     this.selectRow = this.selectRow.bind(this);
     this.generateDataSource = this.generateDataSource.bind(this);
-
+    
+    const { selectedOptions } = props;
     this.state = {
-      selectedRows: [],
+      selectedRows: selectedOptions || [],
```
Comment (robcalcroft) [inline MultiSelect.js]: Here, could you just use `this.props.selectedOptions` please; pulling out the single prop looks a little untidy. Other than that happy to merge 😄

**Comment by samschooler** [comment]
I fixed that! Should be good! Thanks

**Comment by robcalcroft** [comment]
Will push to NPM now 😄

**Comment by robcalcroft** [comment]
Thanks for the contrib 👍

**Pull request merged**